VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:


BO_ 300 Message_C: 8 Vector__XXX
 SG_ Signal_Sine_Shifted : 0|64@1- (1,0) [-1.7E+308|1.7E+308] "V" Vector__XXX

BO_ 400 Message_D: 2 Vector__XXX
 SG_ Signal_Triangle : 8|8@1- (1,0) [-128|127] "" Vector__XXX
 SG_ Signal_Random : 0|8@1+ (1,0) [0|255] "degC" Vector__XXX

BO_ 200 Message_B: 8 Vector__XXX
 SG_ Signal_Sine : 0|64@1- (1,0) [-1.7E+308|1.7E+308] "V" Vector__XXX

BO_ 100 Message_A: 4 Vector__XXX
 SG_ Signal_Step_Counter : 16|16@1+ (1,0) [0|65535] "N" Vector__XXX
 SG_ Signal_PWM : 0|16@1+ (1,0) [0|65535] "N" Vector__XXX



BA_DEF_ BO_  "CANFD_BRS" ENUM  "0","1";
BA_DEF_  "DBName" STRING ;
BA_DEF_  "BusType" STRING ;
BA_DEF_ BU_  "NodeLayerModules" STRING ;
BA_DEF_ BU_  "ECU" STRING ;
BA_DEF_ BU_  "CANoeJitterMax" INT 0 0;
BA_DEF_ BU_  "CANoeJitterMin" INT 0 0;
BA_DEF_ BU_  "CANoeDrift" INT 0 0;
BA_DEF_ BU_  "CANoeStartDelay" INT 0 0;
BA_DEF_ BO_  "VFrameFormat" ENUM  "StandardCAN","ExtendedCAN","reserved","reserved","reserved","reserved","reserved","reserved","reserved","reserved","reserved","reserved","reserved","reserved","StandardCAN_FD","ExtendedCAN_FD";
BA_DEF_DEF_  "CANFD_BRS" "1";
BA_DEF_DEF_  "DBName" "";
BA_DEF_DEF_  "BusType" "";
BA_DEF_DEF_  "NodeLayerModules" "";
BA_DEF_DEF_  "ECU" "";
BA_DEF_DEF_  "CANoeJitterMax" 0;
BA_DEF_DEF_  "CANoeJitterMin" 0;
BA_DEF_DEF_  "CANoeDrift" 0;
BA_DEF_DEF_  "CANoeStartDelay" 0;
BA_DEF_DEF_  "VFrameFormat" "StandardCAN";
BA_ "BusType" "CAN";
BA_ "DBName" "CANExplorerDatabase";
SIG_VALTYPE_ 300 Signal_Sine_Shifted : 2;
SIG_VALTYPE_ 200 Signal_Sine : 2;

